ftplugin/ps1.vim

```diff
@@ -42,10 +42,4 @@ function! s:PSESSetup ()
             \)
 endfunction
 
-" Set the file type to ps1 for ps1, psm1, and psd1
-" 'ps1' is what vim-polyglot uses for styling
-if(&filetype == "")
-    autocmd BufNewFile,BufRead *.ps*1 set filetype=ps1
-endif
-
-autocmd FileType ps1,psd1,psm1 call s:PSESSetup()
+call s:PSESSetup()
```

Comment:
Should this run every time the user opens a powershell file?

Reply:
no, only the first time, so if ftdetect is done properly we just need to fire it once. 👍

Reply:
```vim
if exists('s:loaded_ftplugin')
  finish
endif
let s:loaded_ftplugin = 1
call s:PSESetup()
```
Treat it like a plugin then with a guard?

Reply:
yup. that will do.